{"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"tokenNotExist"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"}],"metadata":{"roleHeading":"Case","externalID":"s:9VigramSDK19AuthenticationErrorO13tokenNotExistyACSScACmF","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"tokenNotExist"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"role":"symbol","title":"AuthenticationError.tokenNotExist(_:)","modules":[{"name":"VigramSDK"}]},"abstract":[{"type":"text","text":"Error authentication - token is not exist"}],"kind":"symbol","sections":[],"schemaVersion":{"minor":3,"patch":0,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vigramsdk\/authenticationerror\/tokennotexist(_:)"]}],"identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/tokenNotExist(_:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/tokenNotExist(_:)":{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/tokenNotExist(_:)","title":"AuthenticationError.tokenNotExist(_:)","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"tokenNotExist"},{"kind":"text","text":"("},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/tokennotexist(_:)","abstract":[{"text":"Error authentication - token is not exist","type":"text"}],"kind":"symbol","type":"topic"},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"title":"VigramSDK","kind":"symbol","type":"topic","abstract":[],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","url":"\/documentation\/vigramsdk","role":"collection","images":[{"type":"icon","identifier":"logo.png"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"url":"\/documentation\/vigramsdk\/authenticationerror","type":"topic","abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}],"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","title":"AuthenticationError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"AuthenticationError","kind":"identifier"}]},"logo.png":{"type":"image","alt":"Vigram logo.","identifier":"logo.png","variants":[{"traits":["1x","light"],"url":"\/images\/logo.png"}]}}}